deployment/docker-compose/docker-compose.yml

```diff
@@ -7,14 +7,19 @@ services:
     command: >
       /bin/sh startup.sh
     restart: always
+    volumes:
+      - temp:/usr/src/aaq_backend/temp
+      - ./.gcp_credentials.json:/app/credentials.json
     env_file:
       - .base.env
       - .core_backend.env
+      - .litellm_proxy.env
```

Comment:
Why adding `.litellm_proxy.env`

Reply:
so that we can use the `Google cloud storage buckets` and also use these `external apis` without explicitly running the speech container

Reply:
I see. I do believe that if we are using some variables in more than one container we should put it in .base.env. But that's a discussion for another day.